docs/getting-started.md

```diff
@@ -2,20 +2,267 @@
+3. Messages triggered by Notifi-hosted code, usually in response to on-chain events
+
+For this Getting Started guide, we will be creating a Community Manager topic.
+Check our our 
+[self-hosted](./getting-started-with-self-hosted)
+and [Notifi-hosted](./getting-started-with-notifi-hosted)
+guides for writing for the other two types.
+
+To create a Community Manager topic, go to the Alert Manager, click **Create Topic**,
+then **Send announcements from Community Manager**.
+
+(TODO: Screenshots)
+
+In the dialog box that appears, set the name of your topic and click **Create**.
+
+## Setting Up Your Signup UI
+
+The easiest way to set up signup UI for your users is the Notifi React Card, a
+React component that you can integrate with your application. This has two steps:
+creating the UI Card config in Admin Portal, and adding the card to your application.
+
+### Creating a UI Card Config
+
+To create your UI card configuration, go to the **UI Config** tab in Alert Manager,
+then click **Create UI Card**.
+
+(TODO: Add screenshot)
+
+You'll be presented with the Edit UI Card screen, which will let you edit your card
+and show you a preview.
+
+1. Enter a name for your card.
+2. Set the wallet type that you want to use to connect to the card. For this example,
+   we will use Ethereum (though the instructions will be the same for any EVM chain).
+   - Note that each card config only supports one wallet at a time. If you want to allow
+     signups across multiple blockchains, create a separate card for each.
+3. Click **Select Topics**, then on the dialog box that pops up, click the check box
+   next to the Community Manager topic you created earlier. Click Save to put the
+   Edit UI Card back in focus.
+4. If you wish, add a tooltip to your topic by clicking the **Add Tooltip** button.
+   In the dialog box that appears, type a short note to explain your topic.
+5. Under **Destinations**, click the switches next to the destination types you want
+   to enable. For this Getting Started guide, we will just activate "Email", as that
+   requires the least setup.
+6. Click **Save Card** in the top right corner of the screen.
+
+You're all done! Leave this page open so that you can reference the Card ID when
+you add the card to your website.
+
+(TODO: Screenshots for setting up your card)
+
+### Setting Up a React Application With Your Card
+
+First, either open your existing JSX React project, or create a new one.
+(TODO: Resources for creating a React app)
+
+Once you've decided on the JSX page where you want to put the card, 
+paste these imports to the top of the file:
+
+```
+import {
+  NotifiContext,
+  NotifiSubscriptionCard,
+} from '@notifi-network/notifi-react-card';
+import '@notifi-network/notifi-react-card/dist/index.css';
+```
+
+and paste this code at the exact spot where you want your card to be:
+```
+<NotifiContext
+    dappAddress="<YOUR OWN DAPP ADDRESS HERE>"
+    env="Production"
```

Comment:
Let's make env optional and default to Production

Reply:
Done: https://github.com/notifi-network/notifi-sdk-ts/pull/537

Reply:
I'm going to leave it in until we have a release that actually allows `env` to be optional, though.